resources/definitions/weedo_base.def.json

```diff
@@ -0,0 +1,448 @@
+    },
+    "initial_layer_line_width_factor": {
+      "default_value": 110.0
+    },
+    "wall_0_wipe_dist": {
+      "value": 0.0
+    },
+    "z_seam_corner": {
+      "default_value": "z_seam_corner_any"
+    },
+    "skin_outline_count": {
+      "value": "0"
+    },
+    "infill_sparse_density": {
+      "default_value": 10.0
+    },
+    "infill_pattern": {
+      "value": "'zigzag'"
+    },
+    "infill_before_walls": {
+      "default_value": false
+    },
+    "material_print_temperature": {
+      "maximum_value": "300"
+    },
+    "material_print_temperature_layer_0": {
+      "value": "min(material_print_temperature+10,300)"
+    },
+    "material_initial_print_temperature": {
+      "value": "material_print_temperature"
+    },
+    "material_final_print_temperature": {
+      "value": "material_print_temperature"
+    },
+    "material_bed_temperature": {
+      "maximum_value_warning": "96"
+    },
+    "material_flow": {
+      "default_value": 95.0
+    },
+    "material_flow_layer_0": {
+      "default_value": 95.0
+    },
+    "infill_material_flow": {
+      "value": "max(0,material_flow-5)"
+    },
+    "support_material_flow": {
+      "value": "max(0,material_flow-5)"
+    },
+    "speed_print": {
+      "default_value": 70.0
+    },
+    "speed_infill": {
+      "default_value": 70.0
+    },
+    "speed_wall": {
+      "value": "max(5,round(speed_print / 2-5,1))"
+    },
+    "speed_wall_0": {
+      "value": "max(5,speed_wall-5)"
+    },
+    "speed_wall_x": {
+      "value": "speed_wall"
+    },
+    "speed_topbottom": {
+      "value": "round(speed_print * 0.65,1)"
+    },
+    "speed_support": {
+      "value": "round(speed_print*0.82,1)"
+    },
+    "speed_support_infill": {
+      "value": "speed_support"
+    },
+    "speed_support_interface": {
+      "value": "round(speed_support*0.689,1)"
+    },
+    "speed_prime_tower": {
+      "value": "speed_support"
+    },
+    "speed_travel": {
+      "value": 105.0
+    },
+    "speed_layer_0": {
+      "value": 26.0
+    },
+    "speed_print_layer_0": {
+      "value": 26.0
+    },
+    "speed_travel_layer_0": {
+      "value": 80.0
+    },
+    "skirt_brim_speed": {
+      "value": 26.0
+    },
+    "retraction_amount": {
+      "default_value": 3
+    },
+    "retraction_speed": {
+      "default_value": 28.0
+    },
+    "retraction_min_travel": {
+      "default_value": 0.8
+    },
+    "retraction_extrusion_window": {
+      "value": 1.0
+    },
+    "material_standby_temperature": {
+      "default_value": 175.0
+    },
+    "switch_extruder_retraction_amount": {
+      "value": 16.5
+    },
+    "switch_extruder_retraction_speeds": {
+      "default_value": 28.0
+    },
+    "retraction_combing": {
+      "value": "off"
+    },
+    "retraction_hop_after_extruder_switch": {
+      "default_value": false
+    },
+    "support_angle": {
+      "default_value": 60.0
+    },
+    "support_pattern": {
+      "default_value": "zigzag"
```

Comment:
That is already the default value defined in fdmprinter. Try to adhere to the DRY (Do not Repeat Yourself) principle as much as possible. This will make maintaining the profile a lot easier

Reply:
Thanks for the reviews. I will update the profiles according to all suggestions and commit again.